cats-effect/src/main/scala/org/specs2/cats/effect/SyncIOMatchers.scala

```diff
@@ -22,9 +23,11 @@ trait SyncIOMatchers extends ValueChecks:
         }
         .unsafeRunSync()
 
+  @targetName("syncIOBeSuccess")
   def beSuccess[A]: Matcher[SyncIO[A]] =
```

Comment:
`beSuccess` is defined also for `IOMatchers` and they are both inherited in `trait CatsEffect`. Using the `@targetName` annotation disambiguates them.

Reply:
https://dotty.epfl.ch/docs/reference/other-new-features/targetName.html#relationship-with-overriding